library/src/main/scala/scala/scalajs/js/package.scala

```diff
@@ -192,6 +192,22 @@ package object js {
    *  not be nested in any local method, class, by-name argument or closure.
    *  The latter includes `for` comprehensions. They may appear within
    *  conditional branches, `while` loops and `try/catch/finally` blocks.
+   *
+   *  <h2>Orphan `await`s in WebAssembly</h2>
+   *
+   *  When compiling for Scala.js-on-Wasm only, you can allow calls to
+   *  `js.await` anywhere, by adding the following import:
+   *
+   *  {{{
+   *  import scala.scalajs.js.wasm.JSPI.allowOrphanJSAwait
+   *  }}}
+   *
+   *  Calls to orphan `js.await`s are validated at run-time. There must exist
+   *  a dynamically enclosing `js.async { ... }` block on the call stack.
+   *  Moreover, there cannot be any JavaScript frame (function invocation) in
```

Comment:
```suggestion
   *  Moreover, there cannot be any JavaScript frame (JavaScript function invocation) in
```

Just saw this. Otherwise, one might interpret "JavaScript frame" = "function invocation" which is wrong (and would make the feature useless).